coalib/bearlib/languages/__init__.py

```diff
@@ -3,39 +3,38 @@
 """
 
 # Start ignoring PyUnusedCodeBear
-from .Language import Language
-from .Language import Languages
-
-from .definitions.Unknown import Unknown
 from .definitions.Bash import Bash
 from .definitions.C import C
 from .definitions.CPP import CPP
-from .definitions.CSharp import CSharp
 from .definitions.CSS import CSS
+from .definitions.CSharp import CSharp
 from .definitions.Fortran import Fortran
 from .definitions.Golang import Golang
 from .definitions.html import HTML
-from .definitions.Java import Java
-from .definitions.JavaScript import JavaScript
 from .definitions.JSON import JSON
 from .definitions.JSP import JSP
+from .definitions.Java import Java
```

Comment:
@Mah1ndra ```Java``` should be above ```JSON``` alphabetically. (Are you considering Upper-case to be higher than Lower-case?)